wackyracesV1.3/wackyraces.vbs.patch

```diff
@@ -0,0 +1,27 @@
+--- wackyraces.vbs.original	2023-11-22 21:07:57.925144601 +0100
++++ wackyraces.vbs	2023-11-22 22:49:33.733488750 +0100
+@@ -2829,7 +2829,7 @@
+                 DMDScene.GetImage("Dig" & i).SetBounds 4 + i * 6, 3 + 16 + 2, 8, 8
+             Next
+             For i = 20 to 35 ' Bottom
+-                DMDScene.GetImage("Dig" & i).SetBounds (i - 20) * 8, 3, 8, 16
++                DMDScene.GetImage("Dig" & i).SetBounds 8 * (i - 20), 3, 8, 16
+             Next
+             FlexDMD.LockRenderThread
+             FlexDMD.Stage.AddActor DMDScene
+@@ -4008,7 +4008,7 @@
+ 
+ sub BurnOut_hit()
+ 	vpmtimer.addtimer 200,"fLASHBURNOUT '"
+-	vpmtimer.addtimer 330,"fLASHBURNOUT '"
++	vpmtimer.addtimer 330,"fLASHBURNOUT '"
+ 	vpmtimer.addtimer 400,"fLASHBURNOUT '"
+ 	vpmtimer.addtimer 450,"fLASHBURNOUT '"
+ 
+@@ -13058,4 +13058,4 @@
+ 
+ '*******************************************
+ '  End Ramp Triggers
+-'*******************************************
+\ No newline at end of file
++'*******************************************

```

Comment:
Could also be reverted?

Reply:
this one neither. It's so strange..

Reply:
Could be your editor that does this automatically